src/context/ContextCalendar.jsx

```diff
@@ -9,12 +16,41 @@ class ContextCalendar extends React.Component {
         this.props.onSelected(this.props.state, context);
     }
 
+    onKeyDown = (e, data, state, editor) => {
+        const keyCode = e.which;
+        if (keyCode === ENTER_KEY) {
+            e.preventDefault();
+            e.stopPropagation();
+            if (_.isUndefined(this.refs.input.state.selectedValue)) {
+                this.refs.input.state.selectedValue = this.refs.input.state.value;
+            }
+            this.handleDateSelect(this.refs.input.state.selectedValue);
+        } else if (keyCode === UP_ARROW_KEY) {
+            e.preventDefault();
+            e.stopPropagation();
+            this.refs.input.goTime(-1, 'weeks');
+        } else if (keyCode === DOWN_ARROW_KEY) {
+            e.preventDefault();
+            e.stopPropagation();
+            this.refs.input.goTime(1, 'weeks');
+        } else if (keyCode === LEFT_ARROW_KEY) {
+            e.preventDefault();
+            e.stopPropagation();
+            this.refs.input.goTime(-1, 'days');
+        } else if (keyCode === RIGHT_ARROW_KEY) {
+            e.preventDefault();
+            e.stopPropagation();
+            this.refs.input.goTime(1, 'days');
+        }
+    }
+
+
```

Comment:
[MG2] Can we move the calls to preventDefault and stopPropagation to a function so we don't have to keep calling them individually?